cli/js/web/fetch.ts

```diff
@@ -590,26 +302,55 @@ export async function fetch(
   while (remRedirectCount) {
     const fetchResponse = await sendFetchReq(url, method, headers, body);
 
-    const response = new Response(
+    const responseBody = new ReadableStreamImpl({
```

Comment:
@kitsonk is there a way to do proper cleanup on a `ReadableStream`, haven't found any method / listener to properly close `rid` upon error. If there isn't such a way I'll add a `try/catch/finally` to `pull` method.